restcomm/restcomm.application/src/main/webapp/WEB-INF/scripts/mariadb/init.sql

```diff
@@ -400,6 +400,18 @@ configuration_data LONGTEXT NOT NULL,
 PRIMARY KEY (account_sid, extension_sid)
 );
 
+CREATE TABLE restcomm_permissions(
+sid VARCHAR(34) NOT NULL PRIMARY KEY,
+name LONGTEXT NOT NULL
+);
+
+CREATE TABLE restcomm_accounts_permissions(
+account_sid VARCHAR(34) NOT NULL,
+permission_sid VARCHAR(34) NOT NULL,
```

Comment:
i guess this is foreign key to permissions table...

Reply:
correct, but we dont define any FK constraints